pytensor/tensor/rewriting/basic.py

```diff
@@ -135,6 +135,18 @@ def alloc_like(
     return rval
 
 
+def make_inplace(node, inplace_prop="inplace"):
```

Comment:
Having a hard time understanding this function, so what it looks like it does is: 
- checks if operator is wrapped in Blockwise, pulls it out
- if props["overwrite_a"] is true return false, otherwise set it to true (mostly this step is the confusing one to me)
- reconstruct the op with "overwrite_a" true
- return Apply node for the reconstructed op

Reply:
You got it. I think what's confusing (for me) is that the `False` that's returned if `props["overwrite_a"]` is already `True` is for the rewriter. I guess rewrites either return a new Apply node (if the rewrite is deemed necessary) or False if it's deemed unnecessary.

Reply:
Yup. Return a list/dict of replacement of variables or None/False to indicate rewrite doesn't apply

Reply:
ah okay makes sense now